paasta_tools/utils.py

```diff
@@ -1315,6 +1315,8 @@ def get_readable_files_in_glob(glob: str, path: str) -> List[str]:
         'cluster_fqdn_format': str,
         'chronos_config': ChronosConfig,
         'marathon_config': MarathonConfigDict,
+        'marathon_servers': List[MarathonConfigDict],
+        'previous_marathon_servers': List[MarathonConfigDict],
```

Comment:
Might as well use MarathonServers here since you've defined it.

I think I probably would've just used List[MarathonConfigDict] everywhere though.